hackathon_site/dashboard/frontend/package.json

```diff
@@ -15,7 +15,6 @@
     "formik": "^2.1.4",
     "history": "^4.10.1",
     "lint-staged": "^10.2.2",
-    "node-sass": "^4.14.0",
```

Comment:
we might not merge the Flow one, so I think it would be better to do it here. I only did it in the Flow one because the scss wouldn't compile with node-sass after I installed Flow